src/frontend/screens/Library/index.tsx

```diff
@@ -242,15 +236,13 @@ export default React.memo(function Library(): JSX.Element {
   }
 
   // cache list of games being installed
-  const [installing, setInstalling] = useState<string[]>([])
-
-  useEffect(() => {
-    const newInstalling = libraryStatus
-      .filter((st: GameStatus) => st.status === 'installing')
-      .map((st: GameStatus) => st.appName)
-
-    setInstalling(newInstalling)
-  }, [libraryStatus])
+  const installing = useMemo(
+    () =>
+      libraryStatus
+        .filter((st) => st.status === 'installing')
+        .map((st) => st.appName),
+    [libraryStatus]
+  )
```

Comment:
replaced useState with useMemo to make it simpler

also removed the type since it's already inferred anyway, removes one import